pkg/berglas/berglas_doc_test.go

```diff
@@ -39,12 +40,14 @@ func ExampleNew() {
 }
 
 func ExampleClient_Create() {
-	err = client.Create(ctx, &berglas.CreateRequest{
+	generation, err = client.Create(ctx, &berglas.CreateRequest{
```

Comment:
I'm concerned about changing the function signature. Yes this is < 1.0 so it's technically allowed, but I'd like to avoid if at all possible.

If we have to change it, I'd rather we return a `CreateResponse` struct to future-proof ourselves.

Reply:
I like the `CreateResponse` idea.